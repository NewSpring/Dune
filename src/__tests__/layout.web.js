import renderer from "react-test-renderer";
import Layout from "../layout.web";

const testProps = [
  {
    path: "Counter",
    id: "1",
    Component: () => <h1>Hello World</h1>,
  },
  {
    path: "HelloWorld",
    id: "2",
    Component: () => <p>Hello World Redux</p>,
  },
];

describe("layout web", () => {
  it("works", () => {
    const tree = renderer.create(<Layout components={testProps} />);
    expect(tree).toMatchSnapshot();
  });
});
