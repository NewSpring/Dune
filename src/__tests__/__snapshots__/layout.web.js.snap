// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`layout web works 1`] = `
<div>
  <h1>
    Hello World
  </h1>
  <p>
    Hello World Redux
  </p>
</div>
`;
